packages/docs/src/mdx/coreComponents/Tab.mdx

```diff
@@ -68,6 +68,7 @@ export const VTABS = [
     id: '1',
     label: 'Home',
     icon: HomeIcon,
+    TabBase,
```

Comment:
Do we need this?

Reply:
I don't know? :smile:

Reply:
Removed. Don't know why it ended up there.